devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -203,17 +215,34 @@ private getButtonMap() {[
 				"02" : 3,
 				"03" : 1,
 				"04" : 2
+		] ,
+		"ADUROLIGHT_CSC" : [
+			"01" : 1,
+			"02" : 2,
+			"03" : 3,
+			"04" : 4
+		],
+		"Adurolight_NCC" : [
+			"01" : 1,
+			"02" : 2,
+			"03" : 3,
+			"04" : 4
```

Comment:
the code using `getButtonMap` is not intended to be used in the flow you've outlined I'm pretty sure, making these additions unnecessary.